src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/adjudications/AdjudicationsMigrationService.kt

```diff
@@ -357,7 +359,13 @@ class AdjudicationsMigrationService(
         nomisHearingId = it.oicHearingId,
       )
     } ?: emptyList(),
-    punishments = mapping.punishmentMappings?.map {
+    punishments = mapping.punishmentMappings?.filter {
+      (it.sanctionSeq != null).also { hasSanction ->
+        if (!hasSanction) {
+          log.warn("Ignoring mapping for punishment ${it.punishmentId} for booking ${it.bookingId}. DPS has no sanctionSeq as this is a WIP")
+        }
+      }
+    }?.map {
```

Comment:
just log and filter out where sanctionSeq is null - which really means DPS can't create a punishment for now (but will once the analysis has been done)